administration/playwright.config.ts

```diff
@@ -25,76 +25,22 @@ export default defineConfig({
   reporter: 'html',
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
-    /* Base URL to use in actions like `await page.goto('/')`. */
-    // baseURL: 'http://127.0.0.1:3000',
-
+    baseURL: 'http://localhost:3000',
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: 'on-first-retry',
+    screenshot: 'only-on-failure'
   },
 
   /* Configure projects for major browsers */
   projects: [
     {
       name: 'chromium',
-      testMatch: 'create_applications.spec.ts',
```

Comment:
I guess the idea behind this was to setup the order of the test execution?
The same can be achieved if we add numbers to the file names - they are executed in the alphabetical order.
But:
Actually tests should be independent from each other.
If "create application" fails due to a potential bug - this should not cause all other tests to fail.
In order to be able to test cases that require test data to be created, we must configure the loading of test data.
I will create a task for that.

Reply:
Yes this should keep them in order.

Reply:
I can see delete applications is running in parallel 🫣

Reply:
I had to revert the changes to keep it in order.